src/vs/editor/browser/services/openerService.ts

```diff
@@ -158,22 +130,10 @@ export class OpenerService implements IOpenerService {
 
 		return Promise.resolve(true);
 	}
-}
 
-/**
- * Check whether a domain like https://www.microsoft.com matches
- * the list of trusted domains.
- */
-function isDomainTrusted(domain: string, trustedDomains: string[]) {
-	for (let i = 0; i < trustedDomains.length; i++) {
-		if (trustedDomains[i] === '*') {
-			return true;
-		}
-
-		if (trustedDomains[i] === domain) {
-			return true;
+	dispose() {
```

Comment:
I do not think this is necessary, but if you go here, then also clear the opener Set. If you use a `Map` you can call `map.clear();`